docs/standard/assembly/placement.md

```diff
@@ -11,13 +11,13 @@ author: "rpetrusha"
 ms.author: "ronpet"
```

Comment:
I hate to ask, Theano, but in looking at this topic, it really should stay in the framework documentation, rather than standard. If it's moved here, we should add a section on placement for Core, but that doesn't strike me as being particularly useful, since the build tools handle most of the details. Would you mind moving it back?

Reply:
@rpetrusha - No problem, will do this shortly.